aspnetcore/mvc/razor-pages/index.md

```diff
@@ -256,9 +256,9 @@ Add a *_ViewImports.cshtml* file:
 @addTagHelper "*, Microsoft.AspNetCore.Mvc.TagHelpers"
 ```
 
-The `@namespace` directive is a new feature that controls the namespace of the generated code allowing us to get rid of `@using` directives. If the `@namspace` directive is used directly in the page, the page will be put directly in the specified namespace. If the `@namespace` directive is inside a *_ViewImports.cshtml*, the specified namespace is only the prefix. The suffix is the dot-seperated relative path between the folder containing *_ViewImports.cshtml* and the folder containing the page.
+The `@namespace` directive is a new feature that controls the namespace of the generated code.  The `@namespace` directive replaces  `@using` directives. If the `@namspace` directive is used explicitly on a page, the page is in scope for the specified namespace. If the `@namespace` directive is contained in *_ViewImports.cshtml*, the specified namespace is only the prefix. The suffix is the dot-seperated relative path between the folder containing *_ViewImports.cshtml* and the folder containing the page.
```

Comment:
I just noticed I wrote `@namspace` instead of `@namespace`.

I don't like saying the `@namespace` directive "replaces" `@using` directives. I see how that's implied by my original wording, but that's not what I meant. It's true that it allows for some `@using` directives to be removed, but you'll almost always still need `@using`s somewhere.

I wrote "If the `@namspace` [sic] directive is used directly in the page, the page will be put directly in the specified namespace" to explain that the specified namespace is the *entire* namespace if it's defined in the page, not just the prefix as is the case if the namespace is specified in _ViewImports.cshtml. My original wording didn't make this clear either, but I think the new wording is ambiguous because `A.B` could be considered to be "in the scope for (of?)" `A`.